src/paperoni/__main__.py

```diff
@@ -126,31 +125,20 @@ class Refine:
     # [action: append]
     link: list[str]
 
+    # Tags to refine
```

Comment:
Note: you can add `# [action: append]` as a comment to make the interface work like `--tag x --tag y` instead of `--tag x y`